Source/Tests/LinuxSyscalls/Syscalls/Info.cpp

```diff
@@ -19,6 +19,23 @@
 
 namespace FEX::HLE {
   void RegisterInfo() {
+    REGISTER_SYSCALL_IMPL(uname, [](FEXCore::Core::InternalThreadState *Thread, struct utsname *buf) -> uint64_t {
+      struct utsname Local{};
+      if (::uname(&Local) == 0) {
+        strncpy(buf->nodename, Local.nodename, sizeof(Local.nodename));
```

Comment:
From a readability perspective, it might be smarter to just do a memcpy here. 

It will stop people who read this code from wondering if strncpy does the right thing here, or hit's it's edge case.  

It does, but only because the uname syscall will have correctly null-terminated the original string.

Reply:
Fixed